docs/content/how-to/distribute-hosted-cluster-workloads.md

```diff
@@ -28,6 +28,41 @@ In addition:
       role.kubernetes.io/infra: "" 
 ```
 
+## Scheduling Topology Options 
+
+Cluster Service Providers may choose how hosted control planes are isolated or co-located. The three different options are:
+
+ - Shared Everything
+ - Shared Nothing 
+ - Dedicated Request Serving
+
+**NOTE**: Each hosted control plane can run Single Replica or Highly Available. If Highly Available, the control plane will be spread across failure domains via `topology.kubernetes.io/zone` as the topology key.
+
+### Shared Everything
+
+- All hosted control plane pods are scheduled to any node that can run hosted control plane workloads.
```

Comment:
It might be worth mentioning that for this topology you can still use taints/labels to allocate nodes only for control plane workloads with `hypershift.openshift.io/control-plane: true`